axelrod/strategies/titfortat.py

```diff
@@ -636,3 +636,31 @@ def strategy(self, opponent: Player) -> Action:
 
         # Otherwise play previous move
         return self.history[-1]
+
+@FinalTransformer((D,) name_prefix=None)
```

Comment:
You now have a syntax error, type exactly this:

```
@FinalTransformer((D,), name_prefix=None)
```

There's information on running the tests (that would have picked this up immediately) here: http://axelrod.readthedocs.io/en/stable/tutorials/contributing/running_tests.html